src/collector/rrdp.rs

```diff
@@ -1628,6 +1646,34 @@ impl TryFrom<IVec> for RepositoryObject<Bytes> {
 }
 
 
+//------------ FallbackTime --------------------------------------------------
+
+/// Parameters for calculating the best-before time of repositories.
+#[derive(Clone, Copy, Debug)]
+struct FallbackTime {
+    min: Duration,
+    max: Duration,
+}
+
+impl FallbackTime {
+    /// Creates a new value from the configuration.
+    pub fn from_config(config: &Config) -> Self {
+        FallbackTime {
+            min: config.refresh,
+            max: cmp::min(2 * config.refresh, config.rrdp_fallback_time)
```

Comment:
```suggestion
            max: cmp::max(2 * config.refresh, config.rrdp_fallback_time)
```

Reply:
Using `min` would allow a `config.rrdp_fallback_time` lower than `2 * config.refresh`, which is not what you want according to the documentation